Sources/HipMobileUI/Views/LoginScreenView.xaml

```diff
@@ -25,25 +25,32 @@
         <views:NotificationExtensionView>
             <views:NotificationExtensionView.ContentTemplate>
                 <Grid x:Name="Grid" Padding="10" HorizontalOptions="Fill">
-                    <Entry x:Name="EmailEntry" Keyboard="Email" Placeholder="{helpers:Translate LoginScreenView_Email}"
-                           Text="{Binding Email}" TextChanged="Entry_TextChanged" />
-                    <Entry x:Name="PasswordEntry" IsPassword="True" Placeholder="{helpers:Translate LoginScreenView_Password}"
+                    <Entry x:Name="EmailEntry"
+                           Keyboard="Email"
+                           VerticalOptions="End"
+                           Placeholder="{helpers:Translate LoginScreenView_Email}" Text="{Binding Email}" TextChanged="Entry_TextChanged" />
+                    <Entry x:Name="PasswordEntry"
+                           IsPassword="True" Placeholder="{helpers:Translate LoginScreenView_Password}"
                            Text="{Binding Password}" TextChanged="Entry_TextChanged" />
-                    <Label x:Name="ErrorMessageLabel" FontSize="Small" HorizontalOptions="Center" TextColor="Red"
-                           Text="{Binding ErrorMessage, Mode=TwoWay}" />
-                    <Label x:Name="RegisterLabel" FontSize="Small" HorizontalOptions="Center" TextColor="Blue"
-                           Text="{helpers:Translate LoginScreenView_Register}">
+                    <Label x:Name="ErrorMessageLabel"
+                           HorizontalOptions="Center" VerticalOptions="Start"
+                           FontSize="Small" TextColor="Red" Text="{Binding ErrorMessage, Mode=TwoWay}" />
+                    <Label x:Name="RegisterLabel"
+                           HorizontalOptions="Center"
+                           FontSize="Small" TextColor="Blue" Text="{helpers:Translate LoginScreenView_Register}">
                         <Label.GestureRecognizers>
                             <TapGestureRecognizer Command="{Binding RegisterCommand}" />
                         </Label.GestureRecognizers>
                     </Label>
-                    <Label x:Name="ForgotPasswordLabel" FontSize="Small" HorizontalOptions="Center" TextColor="Blue"
-                           Text="{helpers:Translate LoginScreenView_Forgot_Password}">
+                    <Label x:Name="ForgotPasswordLabel"
+                           HorizontalOptions="Center"
+                           FontSize="Small" TextColor="Blue" Text="{helpers:Translate LoginScreenView_Forgot_Password}">
```

Comment:
The text color for these labels was defined as _blue_ before. You are right, we should use the colors from the App.xaml-file since it's a not-neutral color. But this change will be part of a upcoming task to re-style the entire application.